lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/BltiEntity.java

```diff
@@ -368,7 +368,7 @@ public int getSubmissionCount(String user) {
 	else if (appStoresOnly) {
```

Comment:
can you use brackets above and then put the IF statement in a parentheses?

Reply:
Sure thing - that was David's code - happy to clean it up